webui/src/app/components/graphs/graph-browse.component.ts

```diff
@@ -12,6 +12,7 @@ import { GraphBean, NodeBean, GraphVis } from '../../models/graph/graph-bean';
 import { DataDeploymentService } from '../../services/data-deployment.service';
 import { AutoUnsubscribe } from '../../shared/decorator/autoUnsubscribe';
 import { GraphComponent } from '../../widget/graph/graph.component';
+import { Observable } from 'rxjs/internal/Observable';
```

Comment:
Sounds very weird to rely on someone else' internals! Auto-import typo for `rxjs/Observable`?